internal/db/model/setup.go

```diff
@@ -16,6 +16,7 @@ import (
 const (
 	FinalityProviderDetailsCollection = "finality_provider_details"
 	BTCDelegationDetailsCollection    = "btc_delegation_details"
+	TimeLockCollection                = "timelock_queue"
```

Comment:
```suggestion
	TimeLockCollection                = "timelock"
```
what does `queue` mean?